MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include requirements.txt
+include requirements_mqtt.txt
```

Comment:
Once you don't read the requirements from these files anymore, it's unnecessary to include them in the sdist.